pkg/plugins/models.go

```diff
@@ -74,6 +74,8 @@ type PluginBase struct {
 	GrafanaNetHasUpdate bool   `json:"-"`
 
 	Root *PluginBase
+
+	Errors *PluginErrors
```

Comment:
Why the wrapping struct, which doesn't bring any benefit that I can tell, instead of:

```suggestion
	Errors []PluginError
```

Reply:
Yeah no benefit yet. We're discussing more in detail about what info will be required so will keep this as is for now, but if nothing changes in requirements I'll simplify

Reply:
OK, thanks for the clarification.